scully/renderPlugins/customMarkdownOptions.ts

```diff
@@ -0,0 +1,28 @@
+export const customMarkdownOptions = (html: string) => {
+  return html.replace(/\<a[^>]*\>[^<]*\<\/a\>\{[^}]*\}/g, (val, pos) => {
+    const [start, rest] = val.split('{:');
+    const injectStr = rest.slice(0, -1);
+    const [initial, end] = start.split('href=');
+    return `${initial} ${injectStr} href=${end}`;
+  });
+};
+
```

Comment:
I like it. Remove the comments, and would be nice to add a test or two. Even an e2e test would be the best.